test/helper.lua

```diff
@@ -201,9 +201,6 @@ t.before_suite(function()
         wal_dir    = t.datadir,
         memtx_dir  = t.datadir,
         vinyl_dir  = t.datadir,
-        -- vinyl_memory is changed to test test_mvcc_vinyl_tx_conflict.
-        -- TODO: Commented out until tarantool/expirationd#105 is resolved.
-        -- vinyl_memory = 1024,
```

Comment:
It was needed to force select operation perform read operation from disk and yield.
It's worth to see in commit message why disabling of deferred delete changes behaviour.